### DISCUSSION TITLE
How to use [textual run] with a module

### DISCUSSION THREAD

**Discussion by SkwalExe** [Textualize/textual#3642] [Q&A] [answered]
I have an application that is normally ran with `python3 -m my_app` but I need some debugging features.
How could I run the app with `textual run --dev`? For now it doesn't work because the code uses relative imports.

**Comment by davep** [comment]
On method would be to set the `TEXTUAL` environment variable to `devtools`

For example, depending on your shell:

```sh
$ TEXTUAL=devtools python3 -m my_app
```

**Reply by SkwalExe** [reply]
It still asks to run with `--dev` to see errors

**Reply by davep** [reply, marked as answer]
I do struggle to remember what thing maps to what, but I think there you'd want `TEXTUAL=devtools,debug`.

**Reply by SkwalExe** [reply]
It worked, is this documented somewhere? I didn't find it in the documentation.

**Reply by davep** [reply]
I don't believe it is currently; I'm not sure if it's because it's considered "undocumented", or if it's simply because it hasn't been documented.
